src/futils.cpp

```diff
@@ -481,6 +481,8 @@ namespace Exiv2 {
             path[l]=0;
             ret = path;
         }
+    #elif defined(__MINGW__)
+        // TODO : see https://github.com/Exiv2/exiv2/issues/610
```

Comment:
Are you sure this actually fixes the issue and just doesn't hide the fact that the build is broken?

Reply:
It fixes #610 in the sense that it fixes the compilation with MXE, and it seems that this is enough for him. But of course it is not fixing the underlying issue that we should solve once #645 is merged. 

I know this is a dirty approach, but IMHO a pragmatic one. We make somebody happy and we avoid never-ending discussions.